### PR TITLE
Feral Human Damage/Description Tweak

### PR DESCRIPTION
**Pull request by souricelle**
#### Summary
Bugfixes "Feral Human Damage/Description Tweak"

#### Purpose of change
Feral humans had extra damage on their monstergun rock throw. This was removed so that they do similar damage to when the player throws rocks. They also had very poor stats, much weaker than zombies, and the axe feral wasn't dealing cut damage. Lastly, the description contained a partial sentence and didn't differentiate between the types of ferals, which is rather important now that the axe feral does a lot of cut damage.

#### Describe the solution
Set feral human melee to 3. Zombies have 4, but zombies are singular mindless instinct where these guys are using the remnants of their former skills. The equipment in the case of these particular three implies they were not skilled fighters pre-cataclysm. I also gave the feral human 8 cut damage and took away one of its regular damage dice. The fire axe is a powerful weapon in the hands of the player. Lastly I changed the pronoun of the human from "it" to "they" to underline the fact that this is still in some sense a person. I think this should be standard for future ferals.

#### Describe alternatives you've considered
Ferals could be weaker or stronger, but these basic ones seem pretty middle of the road. Soldier ferals or survivor ferals or feral old ladies might be a different story.

#### Testing
Spawned some feral humans and fought them with a randomly generated fresh survivor. They are significantly more capable now but can still be handled.

#### Additional context
Tangentially related to discussion in the comments [here.](https://github.com/CleverRaven/Cataclysm-DDA/pull/47996)

### CODE REVIEW

File: data/json/monsters/feral_humans.json [hunk 1]
```diff
@@ -70,9 +71,11 @@
   {
     "id": "mon_feral_human_axe",
     "type": "MONSTER",
+    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This axe-wielding maniac still breathes, but the zombies treat ferals like one of them.",
     "copy-from": "mon_feral_human_pipe",
-    "melee_dice": 3,
+    "melee_dice": 2,
     "melee_dice_sides": 8,
+    "melee_cut": 8,
```
Comment (actual-nh) [inline data/json/monsters/feral_humans.json:78]:
```suggestion
    "melee_dice": 2,
    "melee_dice_sides": 7,
    "melee_cut": 9,
    "extend": { "flags": [ "FILTHY" ] },
```
I calculated out the damage from a fire axe a bit ago. (Filthy is for the blood-stained clothing, as you might guess.)

File: data/json/monsters/feral_humans.json [hunk 2]
```diff
@@ -70,9 +71,11 @@
   {
     "id": "mon_feral_human_axe",
     "type": "MONSTER",
+    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This axe-wielding maniac still breathes, but the zombies treat ferals like one of them.",
```
Comment (actual-nh) [inline data/json/monsters/feral_humans.json:74]:
```suggestion
    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This axe-wielding maniac with clothes covered in dried blood still breathes, but the zombies treat ferals like one of them.",
```

**Review by actual-nh — COMMENTED** [review]
Alternatively, remove the description alterations for now and add them in a separate PR so this one could go in prior to string freeze.

File: data/json/monsters/feral_humans.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "id": "mon_feral_human_pipe",
     "type": "MONSTER",
     "name": { "str": "feral human" },
-    "description": "Pupils dilated and what remains to be seen of the iris and sclera are bloodshot.  It still breathes but the zombies treat it like one of them.",
+    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This pipe-wielding maniac still breathes, but the zombies treat ferals like one of them.",
```
Comment (anothersimulacrum) [inline data/json/monsters/feral_humans.json:6]: I think it's better for the description to describe how it is treated, not all others like it.
```suggestion
    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This pipe-wielding maniac still breathes, but the zombies treat it like one of them.",
```

File: data/json/monsters/feral_humans.json [hunk 4]
```diff
@@ -62,6 +62,7 @@
   {
     "id": "mon_feral_human_crowbar",
     "type": "MONSTER",
+    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This crowbar-wielding maniac still breathes, but the zombies treat ferals like one of them.",
```
Comment (anothersimulacrum) [inline data/json/monsters/feral_humans.json:65]:
```suggestion
    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This crowbar-wielding maniac still breathes, but the zombies treat it like one of them.",
```

File: data/json/monsters/feral_humans.json (hunk 2 above)
Comment (anothersimulacrum) [reply data/json/monsters/feral_humans.json:74]:
```suggestion
    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This axe-wielding maniac with clothes covered in dried blood still breathes, but the zombies treat it like one of them.",
```
```suggestion
    "description": "Their pupils are dilated and what remains to be seen of the iris and sclera are bloodshot.  This axe-wielding maniac still breathes, but the zombies treat it like one of them.",
```

File: data/json/monsters/feral_humans.json (hunk 3 above)
Comment (souricelle) [reply data/json/monsters/feral_humans.json:6]: Mixing pronouns is confusing. You can't be a they one sentence and an it the next.

**Pull request merged**